### DISCUSSION TITLE
Glob Import Query Params

### DISCUSSION THREAD

**Discussion by simeonradivoev** [parcel-bundler/parcel#9030] [Q&A]
I can see there are query parameters for single files as shown in https://github.com/parcel-bundler/parcel/pull/4881.
But is there a way to resize a bunch of images using the glob resolver?

**Comment by mischnic** [comment]
See https://github.com/parcel-bundler/parcel/issues/8623

**Comment by simeonradivoev** [comment]
So it's not currently possible.
